syntax = "proto3";
import "common.proto";
package autodrive.workflow.proto;


message TransactionMsg {
  // Header
  MsgHeader header = 1;
  // Error
  Error error = 2;
  // topic
  string topic = 3;
  // frame id
  uint64 data_frame_id = 4;
  // DAG id
  uint32 dag_id = 5;
  // enter DAG time
  uint64 enter_time = 6;
}